common/src/java/com/zimbra/common/util/ZimbraCookie.java

```diff
@@ -47,6 +47,12 @@ public static void setAuthTokenCookieDomainPath(Cookie cookie, String path) {
             cookie.setDomain(LC.zimbra_authtoken_cookie_domain.value());
         }
 
+        String cookieVal = LC.zimbra_same_site_cookie.value();
+        if (!StringUtil.isNullOrEmpty(cookieVal)) {
+            // setting cookie value like "SameSite=Strict;", value can be Strict, Lax, None
+            path = new StringBuilder(path).append(";SameSite=").append(cookieVal).append(";").toString();
```

Comment:
If the `cookieVal` is set to `None`, you will need to ensure that the `Secure` flag is also set. If `Secure` is not set when the `SameSite` value is `None`, the cookie will be blocked. It would be possible to accidentally block all cookies by simply setting the config value to `None`.

May also want to consider the case when cookies are being cleared (value of cooke set to `""`). At a minimum, existing cookies may not be cleared once this change is deployed (and cookie updates restricted).